### PR TITLE
Updated gitignore and removed redundant files

### PR DESCRIPTION
**Pull request by zasviat**
https://stackoverflow.com/questions/3833561/why-doesnt-git-ignore-my-specified-file

Git does NOT ignore files if they're not untracked
So we have to make git forget them forcely using this command
`git rm --cached <path_to_file>`
OR
`git rm --cached -r <path_to_folder>`
-r means recursive run for `rm` command

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,4 @@ __pycache__/
 .env
 logs/
 app/app.db
-requirements.txt
+dump.rdb
```
Comment (zasviat) [inline .gitignore:11]: dump.rdb is Redis cache file so it can be added to .gitignore
In more detail:
https://redis.io/docs/management/persistence/

**Comment by ViktorLitviakov8050** [comment]
interesting, need to discuss it on the lesson.

**Comment by ViktorLitviakov8050** [comment]
already in master

**Comment by ViktorLitviakov8050** [comment]
already in master

**Pull request closed without merging**